rskj-core/src/test/java/org/ethereum/net/client/CapabilityTest.java

```diff
@@ -0,0 +1,41 @@
+package org.ethereum.net.client;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+public class CapabilityTest {
+
+    @Test
+    public void testEquals_OK() {
+        // An object must be equal to itself
+        Capability capability = new Capability("task.1", (byte) 0);
+        assertEquals(capability, capability);
+
+        // Same name and version must make Capabilities to be equal (Including name being null)
+        assertEquals(new Capability("task.1", (byte) 0), new Capability("task.1", (byte) 0));
+        assertEquals(new Capability(null, (byte) 0), new Capability(null, (byte) 0));
+
+        // Capability must be different from null
+        assertNotEquals(new Capability("task.1", (byte) 0), null);
```

Comment:
I'd maybe use assertNotNull

Reply:
In this case I think this is better for the sake of clarity, since we want to be explicit about the method `equals` being called.

Reply:
you're 100% right, my comment was automatic 😅